Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7247,6 +7265,2013 @@ def panorama_install_file_content_update_command(args: dict):
         return_results(result['response']['msg'])
 
 
+"""
+PAN-OS Network Operations Integration
+
+Provides additional complex commands for PAN-OS firewalls and ingests configuration issues as incidents.
+"""
+
+
+# Errors
+class OpCommandError(Exception):
+    pass
+
+
+# Best practices
+class BestPractices:
+    SPYWARE_ALERT_THRESHOLD = ["medium, low"]
+    SPYWARE_BLOCK_SEVERITIES = ["critical", "high"]
+    VULNERABILITY_ALERT_THRESHOLD = ["medium, low"]
+    VULNERABILITY_BLOCK_SEVERITIES = ["critical", "high"]
+    URL_BLOCK_CATEGORIES = ["command-and-control", "hacking", "malware", "phishing"]
+
+
+# pan-os-python new classes
+class CustomVersionedPanObject(VersionedPanObject):
+    def __init__(self):
+        super(CustomVersionedPanObject, self).__init__()
+
+    def _refresh_children(self, running_config=False, xml=None):
+        """Override normal refresh method"""
+        # Retrieve the xml if we weren't given it
+        if xml is None:
```

Comment:
So - these have to be like this due to the way xml.etree.Element objects are nullable.

It seems like if an XML element has no text (???) but has child elements, it will return false  when used in a conditional. So, we need to check it's literally a NoneType object.